src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -379,31 +379,31 @@ Some components may not be hidden."</string>
     <string name="revanced_hide_navigation_label_summary_off">Navigation label is shown.</string>
     <string name="revanced_switch_create_with_notifications_button_title">Switch create with notifications</string>
-    <string name="revanced_switch_create_with_notifications_button_summary">"Switch the positions of the create button and notification button by spoofing device information.
+    <string name="revanced_switch_create_with_notifications_button_summary">"Swaps the positions of the create button and notification button by spoofing device information.
 
-• Even if you change this setting, it may not take effect until you reboot the device.
+• If this setting is changed, the device may need to be rebooted for it to take effect.
 • Disabling this setting loads more ads from the server side.
 • You should disable this setting to make video ads visible."</string>
 
     <!-- PreferenceScreen: General, PreferenceCategory: General, PreferenceScreen: Settings menu -->
     <string name="revanced_preference_screen_settings_munu_title">Settings menu</string>
-    <string name="revanced_preference_screen_settings_munu_summary">Hide elements in YouTube settings menu.</string>
+    <string name="revanced_preference_screen_settings_munu_summary">Hide elements of the YouTube settings menu.</string>
 
     <string name="revanced_hide_settings_menu_title">Hide YouTube settings menu</string>
     <string name="revanced_hide_settings_menu_summary">Hide elements in YouTube settings menu.</string>
     <string name="revanced_hide_settings_menu_filter_strings_title">YouTube settings menu filter</string>
-    <string name="revanced_hide_settings_menu_filter_strings_summary">List of YouTube settings menu names to filter separated by a new line.</string>
+    <string name="revanced_hide_settings_menu_filter_strings_summary">List of YouTube settings menu names to filter, separated by new lines.</string>
     <string name="revanced_hide_settings_menu_filter_strings_hint">"Account
 Data saving
 Captions"</string>
 
     <!-- PreferenceScreen: General, PreferenceCategory: General, PreferenceScreen: Toolbar -->
     <string name="revanced_preference_screen_toolbar_title">Toolbar</string>
-    <string name="revanced_preference_screen_toolbar_summary">Hide or change components located on the toolbar such as toolbar buttons, search bar, header.</string>
+    <string name="revanced_preference_screen_toolbar_summary">Hide or change components located on the toolbar, such as the search bar, toolbar buttons, and header.</string>
 
     <string name="revanced_change_youtube_header_title">Change YouTube header</string>
     <string name="revanced_change_youtube_header_summary_on">Premium header is enabled.</string>
-    <string name="revanced_change_youtube_header_summary_off">Generic headers is enabled.</string>
+    <string name="revanced_change_youtube_header_summary_off">Generic header is enabled.</string>
```

Comment:
I wasn't sure if the pluralization is deliberate (since iirc YT occasionally has themed headers). 

If it should be kept plural, it should be as follows:

```suggestion
    <string name="revanced_change_youtube_header_summary_off">Generic headers are enabled.</string>
```